src/main/java/SirDuke.java

```diff
@@ -0,0 +1,210 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+import backend.Task.IllegalStartAndEndDateException;
+import backend.ToDoList;
+
+/**
+ * Class that acts as the User Interface.
+ *<p>
+ *     Using the chatbot:
+ *     - To view the list, use the following command:
+ *     list
+ *
+ *      - To add a To Do to the list, use the following format:
+ *     deadline/description
+ *
+ *      - To add a Deadline to the list, use the following format:
+ *     deadline/description/time to be completed by
+ *
+ *      - To add an Event to the list, use the following format:
+ *      event/description/start time/end time
+ *
+ *      - To mark a task as done, use the following format:
+ *      mark/task index
+ *
+ *      - To unmark a task as done, use the following format:
+ *      unmark/task index
+ *
+ *      - To delete a task, use the following format:
+ *      delete/task index
+ *
+ *      - To close the chatbot, use the following command:
+ *      bye
+ *</p>
+ */
+public class SirDuke {
+
+    public static final String HORIZONTAL_LINE =
+            "____________________________________________________________";
+
+    /**
+     * Prints exit message and exits.
+     */
+    public static void sayBye() {
+        System.out.println(HORIZONTAL_LINE + "\n"
+                + "Godspeed.\n"
+                + HORIZONTAL_LINE);
+        System.exit(0);
+    }
+
+    /**
+     * Prints welcome message and allows the SirDuke chatbot to start receiving commands.
+     * Reads user input from console and interprets them as commands.
+     * Executes commands accordingly.
+     * Updates and saves to listFile only if a command is SUCCESSFULLY EXECUTED.
+     *
+     * @param listFile the file for the contents of the ToDoList to be written into
```

Comment:
The sentences should start with capitalisation, here I am referring to https://nus-cs2103-ay2425s2.github.io/website/schedule/week3/topics.html#implementation-documentation-tools-javadoc-what as a reference.
This is found in the other files as well.

```suggestion
     * @param listFile The file for the contents of the ToDoList to be written into.
```